backend/sample.env

```
@@ -1,5 +1,6 @@
 DEBUG=true 
 LISTENADDR=:8080
+PARALLELTRANSCODES=2
```

```suggestion
PARALLEL_TRANSCODES=2
```
A little more readable

---

Adding an underscore would cause the `dotenv` package we use to read in environment variables to expect `TRANSCODES` to be a variable in a `PARALLEL` struct. We can add an override alias for this if you'd like though.